src/main/java/seedu/address/model/Model.java

```diff
@@ -128,4 +129,15 @@
 
     double getCap();
 
+    //=========== Deadline ==================================================================================
+
+    /**
+     * Adds deadline.
+     */
+    void addDeadline(Deadline deadline);
```

Comment:
this abstract class is being inherited by ModelManager, make sure ModelManager override this method